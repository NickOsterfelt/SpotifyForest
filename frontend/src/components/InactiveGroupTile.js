import React from "react";


//A simple component which is used in GroupDisplay for inactive tiles.
function InactiveGroupTile() {
    return (
        <td className="placeholder-tile">
            <div className="item"></div>
        </td>
    );
}

export default InactiveGroupTile;