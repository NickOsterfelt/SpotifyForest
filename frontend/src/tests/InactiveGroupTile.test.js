
import React from "react";
import { render } from "@testing-library/react";
import InactiveGroupTile from "../components/InactiveGroupTile";

it("renders without crashing", function () {
    render(
        <table>
            <tbody>
                <tr>
                    <InactiveGroupTile />
                </tr>
            </tbody>
        </table>
    );
});

it("matches snapshot", function () {
    const { asFragment } = render(
        <table>
            <tbody>
                <tr>
                    <InactiveGroupTile />
                </tr>
            </tbody>
        </table>
    );
    expect(asFragment()).toMatchSnapshot();
});
