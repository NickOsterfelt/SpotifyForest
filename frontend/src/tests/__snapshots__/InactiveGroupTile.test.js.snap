// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <table>
    <tbody>
      <tr>
        <td
          class="placeholder-tile"
        >
          <div
            class="item"
          />
        </td>
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;
